### PR TITLE
November 2019 test-fix ready for review and QC

### PR DESCRIPTION
**Pull request by kylebunting**
The November 2019 test-fix is ready for review and QC.

### CODE REVIEW

File: Hands-on lab/Before the HOL.md [hunk 1]
```diff
@@ -139,23 +149,145 @@ In this task, you will provision an Azure Linux VM, which will serve as your Jen
...
+
+### Task 4: Provision Cosmos DB using the MongoDB API
+
+In this task, you provision a new Azure Cosmos DB account using the MongoDB API.
+
+1. In the [Azure portal](https://portal.azure.com/), select the **Show portal menu** icon and then select **+Create a resource** from the menu.
+
+    ![The Show portal menu icon is highlighted, and the portal menu is displayed. Create a resource is highlighted in the portal menu.](media/create-a-resource.png "Create a resource")
+
+2. Enter "cosmos" into the Search the Marketplace box, select **Azure Cosmos DB** in the search results, and then select **Create**.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL.md]: Instructions state enter "cosmos" - your screenshot shows "cosmos db" entered into the search line....does this matter?

File: Hands-on lab/Before the HOL.md [hunk 2]
```diff
@@ -54,37 +58,39 @@ Microsoft and the trademarks listed at <https://www.microsoft.com/en-us/legal/in
...
+In this task, you provision a Linux virtual machine (VM) running Ubuntu Server 16.04 LTS, which is used as your development machine throughout this lab. The VM is created using an Azure Resource Manager (ARM) template from a GitHub repository. The ARM template includes a custom extension script that installs Docker, Visual Studio Code (VS Code), MongoDB, and other required software on the VM. The ARM template also adds an inbound port rule that opens port 3389 on the network security group for the VM to allow RDP connections.
 
-> If you would like to review the steps to manually provision the VM and installed software, see [Appendix A](./Appendix-A).
+> **Note**: If you would like to review the steps to provision the VM manually and installed software, see [Appendix A](./Appendix-A).
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL.md]: When I test this link in preview, I get a 404 error.  Will that work as soon as we merge?

File: Hands-on lab/Before the HOL.md (hunk 1 above)
Comment (kylebunting) [reply Hands-on lab/Before the HOL.md]: No, this does not matter. The goal is for them to enter enough to filter down to the correct resource. Both will get them there.

File: Hands-on lab/Before the HOL.md (hunk 2 above)
Comment (kylebunting) [reply Hands-on lab/Before the HOL.md]: It should work, but I went ahead and updated the link to include the file extension. Please let me know if that works.

File: Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md [hunk 3]
```diff
@@ -400,27 +382,28 @@ In this task, you will create the collections needed for your database migration
...
 
     ![The New Collection button is highlighted on the Cosmos DB Collections blade.](media/cosmos-db-new-collection.png "New Collection")
 
-4. On the Add Collection dialog, enter the following:
+5. On the Add Collection dialog, enter the following:
 
     - **Database id**: Select **Use existing** and select the **best-for-you-organics** database from the list.
     - **Collection id**: Enter **users**.
-    - **Storage capacity**: Select **Fixed (10 GB)**.
+    - **Shared key**: Enter **id_**.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md]: Hello!  Instructions say Enter "id_" but the image below shows "_id".

File: Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md [hunk 4]
```diff
@@ -1249,6 +1217,8 @@ In this task, you will use the [Azure App Service Jenkins plugin](https://plugin
 
     ![The best-for-you-build project is highlighted on the Jenkins dashboard.](media/jenkins-dashboard-best-for-you-build.png "Jenkins dashboard")
 
+    > **Note**: You may need to login again using the username **jenkins** and password **Password.1!!**.
+
 4. Select **Configure** from the left-hand menu.
 
     ![Configure is highlighted in the left-hand menu.](media/jenkins-project-configure.png "Jenkins left-hand menu")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md]: Please review line 1234.  The alt-text is "In the Container Registry Access key blade, "  We missed the rest of that sentence.

File: Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md (hunk 3 above)
Comment (kylebunting) [reply Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md]: Fixing with my commit below.

File: Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md (hunk 4 above)
Comment (kylebunting) [reply Hands-on lab/HOL step-by step - OSS PaaS and DevOps.md]: Fixed with the commit below.

**Pull request merged**